### ISSUE TITLE
`Property not found in...` errors not reported

### ISSUE THREAD

**Issue by shauns** [amilajack/eslint-plugin-flowtype-errors#33]
If I have a function

``` js
function foo(bar: {baz: string}) {...}
```

And I call it with

``` js
foo(1234);
```

An error is reported as expected (due to the mismatched type). But if I call it with

``` js
foo({xyz: 'abc'});
```

An error is reported by Flow but not by this plug-in. Seems that the code filtering out issues is incorrect: https://github.com/amilajack/eslint-plugin-flowtype-errors/blob/master/src/collect.js#L76 as `filepath` and `message[0].path` don't match in this case.

**Comment by amilajack** [comment]
@shauns Is there a reason why this kind of error doesn't have a correct filepath? I cant think of any reason why.

**Comment by shauns** [comment]
In the broken case, `filepath` is the caller site, and `message[0].path` is the callee. Suppose it could be a flow responsibility rather than this plug-in? See anything like that elsewhere?

**Comment by shauns** [comment]
I've added a PR with a failing test case for you.

**Comment by amilajack** [comment]
Thanks for this!

**Comment by amilajack** [comment]
Would be great if you could check if `1.5.0` has this issue.

**Comment by jdmota** [comment]
I had the same issue because I was using `strip_root=true` in my `.flowconfig`.

**Comment by amilajack** [comment]
@jdmota what version of the plugin are you using?

**Comment by jdmota** [comment]
@amilajack 2.0.1

**Comment by amilajack** [comment]
> I had the same issue because I was using strip_root=true in my .flowconfig.

If you remove strip_root, does it work?

**Comment by jdmota** [comment]
Yes.

When this line runs: `message[0].path === filepath`,
`message[0].path` is a relative path (because of `strip_root`) but `filepath` is absolute, so it fails.

**Comment by amilajack** [comment]
@jdmota would be you be interested in making a PR for this? I'm a bit busy at the moment studying for interviews 😔

**Comment by jdmota** [comment]
@amilajack I'll see what I can do :)

**Comment by amilajack** [comment]
Reopening because it seems like this issue persists.
